platforms/windows/hax_entry.c

```diff
@@ -431,6 +431,18 @@ NTSTATUS HaxVcpuControl(PDEVICE_OBJECT DeviceObject,
             vcpu_debug(cvcpu, (struct hax_debug_t*)inBuf);
             break;
         }
+        case HAX_VCPU_IOCTL_SET_CPUID: {
+            struct hax_cpuid *cpuid = (struct hax_cpuid *)inBuf;
+            if (inBufLength < sizeof(struct hax_cpuid) +
+                cpuid->total * sizeof(struct hax_cpuid_entry)) {
```

Comment:
This will be a crash, if inBufLength is zero.

Reply:
Good catch. I will update the code immediately. Thanks for your review.